util/lru/lru_test.go

```diff
@@ -40,3 +43,16 @@ func TestLRU(t *testing.T) {
 		t.Errorf("contains 3; should not")
 	}
 }
+
+func BenchmarkLRU(b *testing.B) {
+	const lruSize = 10
```

Comment:
b.ReportAllocs() ?

Reply:
Done:

```
       │ before.txt  │              after.txt              │
       │   sec/op    │   sec/op     vs base                │
LRU-32   67.05n ± 2%   59.73n ± 2%  -10.90% (p=0.000 n=20)

       │ before.txt │             after.txt              │
       │    B/op    │    B/op     vs base                │
LRU-32   21.00 ± 0%   10.00 ± 0%  -52.38% (p=0.000 n=20)

       │ before.txt │           after.txt            │
       │ allocs/op  │ allocs/op   vs base            │
LRU-32   0.000 ± 0%   0.000 ± 0%  ~ (p=1.000 n=20) ¹
```

I don't really understand how we're allocating 10B/op on average, but doing that in 0.000 allocs/op... I guess the way the benchmark is constructed, allocs are infrequent enough that go test rounds it to zero, and so benchstat has no decimals to work with...

But hey, apparently allocating 50% less now!